Tensile/TensileCreateLib/ParseArguments.py

```diff
@@ -46,8 +46,15 @@ def splitExtraParameters(par):
     return (key, value)
 
 
-def parseArguments() -> Dict[str, Any]:
-    """Parse command line arguments for TensileCreateLibrary."""
+def parseArguments(input=None) -> Dict[str, Any]:
+    """Parse command line arguments for TensileCreateLibrary.
+
+    Args:
+        input: List of strings representing command line arguments.
+
+    Returns:
+        A dictionary containing the keys representing options and their values.
+    """
```

Comment:
```suggestion
def parseArguments(input: List[str] | None=None) -> Dict[str, Any]:
    """Parse command line arguments for TensileCreateLibrary.

    Args:
        input: List of strings representing command line arguments. Only used during testing.

    Returns:
        A dictionary containing the keys representing options and their values.
    """
```

~~Since this isn't updated in TensileCreateLibrary main, what's the purpose of adding the `input` parameter?~~

I like this pattern after reading below and seeing it in use. However, it may be good to clarify in the doc-string that this parameter is only used when running tests.

Reply:
Added.